### ISSUE TITLE
"see also" misses a linebreak in text output

### ISSUE THREAD

**Issue by rumpelsepp** [sphinx-doc/sphinx#1793]
I use `make text` to render my documentation. The output of [see also](http://sphinx-doc.org/markup/para.html#directive-seealso) misses a linebreak when using lists:

source

``` rst
.. seealso::

    * http://man7.org/linux/man-pages/man2/socket.2.html
    * http://man7.org/linux/man-pages/man2/setsockopt.2.html
    * http://man7.org/linux/man-pages/man2/bind.2.html
    * https://www.kernel.org/doc/Documentation/networking/can.txt
```

`make text` output

```
See also: * http://man7.org/linux/man-pages/man2/socket.2.html


  * http://man7.org/linux/man-pages/man2/setsockopt.2.html

  * http://man7.org/linux/man-pages/man2/bind.2.html

  * https://www.kernel.org/doc/Documentation/networking/can.txt
```

`make text` should generate:

```
See also: 

  * http://man7.org/linux/man-pages/man2/socket.2.html

  * http://man7.org/linux/man-pages/man2/setsockopt.2.html

  * http://man7.org/linux/man-pages/man2/bind.2.html

  * https://www.kernel.org/doc/Documentation/networking/can.txt
```

**Comment by shimizukawa** [comment]
Thanks for your reporting!
I confirmed, I'll fix it.

**Comment by shimizukawa** [comment]
fixed by #1819
